react/BarContextProvider/index.jsx

```diff
@@ -11,7 +13,7 @@ const BarContextProvider = props => {
         <I18nContext.Provider
           value={{ f: props.f, t: props.t, lang: props.lang }}
         >
-          {props.children}
+          <BreakpointsProvider>{props.children}</BreakpointsProvider>
         </I18nContext.Provider>
       </CozyProvider>
     </Provider>
```

Comment:
At some point we will have a full AppProvider here that could be reused across our apps :)